src/Dsl.php

```diff
@@ -20,7 +20,7 @@ function describe($description, callable $fn)
  */
 function context($description, callable $fn)
 {
-    describe($description, $fn);
+    Context::getInstance()->addSuite($description, $fn);
```

Comment:
Yeah, I probably shouldn't have changed that here. My logic was "less method calls = more faster", especially for PHP versions < 7. In any case, it belongs in its own PR.